src/v2/Apps/FairOrginizer/Components/PastEventsRail.tsx

```diff
@@ -0,0 +1,49 @@
+import { Box, Carousel, Spacer, Text } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { PastEventRailCellFragmentContainer as PastEventRailCell } from "./PastEventRailCell"
+import { PastEventsRail_fairs } from "v2/__generated__/PastEventsRail_fairs.graphql"
+import { extractNodes } from "v2/Utils/extractNodes"
+
+interface PastEventsRailProps {
+  fairs: PastEventsRail_fairs
+}
+
+export const PastEventsRail: React.FC<PastEventsRailProps> = props => {
+  const { fairs } = props
+  const pastFairs = extractNodes(fairs)
+
+  if (pastFairs.length === 0) {
+    return null
+  }
+
+  return (
+    <Box mt={2}>
+      <Text variant="lg" as="h3" mb={4}>
+        Past Events
+      </Text>
+      <Carousel>
```

Comment:
@dzucconi - would you say that `Carousel` is deprecated and that we should be using `Shelf` everywhere now?

Reply:
Yeah, the `Shelf` would be used here. `Carousel` is still useful but has special/particular use-cases now. Though in this case I wonder if this is really just a 4 column grid? There's no specification for the overflow at all and it looks like it adheres to the grid.

Reply:
Looking at the mobile view leads me to believe this is a `Shelf` but there's no scrollbar and we lack the the ability to use grid within a `Shelf`. I left a comment on the Figma. Just using a `Shelf` with a 4:3 aspect ratio image is probably the correct course of action here.

Reply:
Got it. I will use `Shell` and the ratio is 325x244